### PR TITLE
Fix UPGRADE-1.7.md to use code formatting

### PR DESCRIPTION
**Pull request by Djamy**
Am I the first to read this ? :p
Some problems with spacing/tabs so we couldn't see the code formatting properly.

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
Hello @Djamy, thank you for your first contribution.

As I can see, all sub-list items are not rendered as they should: https://github.com/Djamy/pim-community-dev/blob/3d722bf8050fe947d6fdc5167bbd037045614cb6/UPGRADE-1.7.md Can you fix them too please? :)

Mickaël

File: UPGRADE-1.7.md [hunk 1]
```diff
@@ -88,109 +88,107 @@ api:
...
 
-    ```PHP
+```PHP
 Oro\Bundle\UIBundle\OroUIBundle,
 Oro\Bundle\FormBundle\OroFormBundle,
 Pim\Bundle\WebServiceBundle\PimWebServiceBundle,
-    ```
+```
 
     * Add the following bundles in the following functions:
```
Comment (mickaelandrieu) [inline UPGRADE-1.7.md]: this is not rendered correctly

File: UPGRADE-1.7.md [hunk 2]
```diff
@@ -88,109 +88,107 @@ api:
...
     * Remove the following lines:
 
-    ```YAML
+```YAML
 pim_webservice:
         resource: "@PimWebServiceBundle/Resources/config/routing.yml"
-    ```
+```
 
     * Add the following lines:
```
Comment (mickaelandrieu) [inline UPGRADE-1.7.md]: this is not rendered correctly

File: UPGRADE-1.7.md [hunk 3]
```diff
@@ -88,109 +88,107 @@ api:
...
-        ```JSON
+```JSON
     "require": {
             "your/dependency": "version",
             "your/other-dependency": "version",
     }
-        ```
+```
 
     * Then run the command to update your dependencies:
```
Comment (mickaelandrieu) [inline UPGRADE-1.7.md]: this is not rendered correctly

**Comment by Djamy** [comment]
What do you think now ?
I admit, this PR was made a bit too quickly..

**Comment by mickaelandrieu** [comment]
Hi think it's great!

Thank you :)

**Comment by damien-carcel** [comment]
Your PR has been merged @Djamy. Thank you very much for it 👍.

**Pull request merged**